core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -876,6 +872,11 @@ object ConfigCommand extends Config {
         throw new IllegalArgumentException(s"--bootstrap-server must be specified for --all")
       }
 
+      if (options.has(zkTlsConfigFile) && options.has(bootstrapServerOpt)) {
+        throw new IllegalArgumentException("--bootstrap-server doesn't support --zk-tls-config-file option. " +
+          "Please use --zookeeper option instead")
```

Comment:
Is this error only printed for cases that are _not_ supported by `--bootstrap-server`? For example, if both `--bootstrap-server` and `--zk-tls-config-file` are configured and you are trying to do something that is supported by `--bootstrap-server`, the message should suggest removing `--zk-tls-config-file` instead.

Reply:
Good suggestion! Updated the message to:
`--bootstrap-server doesn't support --zk-tls-config-file option. If you are trying to update the ZooKeeper TLS configuration, please use --zookeeper option instead. Otherwise, Please remove --zk-tls-config-file option.`